saleor/core/utils/editorjs.py

```diff
@@ -15,7 +15,24 @@
 }
 
 
-def clean_editor_js(definitions: Optional[Dict], *, to_string: bool = False):
+@overload
+def clean_editor_js(
+    definitions: Union[Dict, str, None], *, to_string: Literal[True]
+) -> str:
+    ...
+
+
+@overload
+def clean_editor_js(definitions: Dict) -> Dict:
+    ...
+
+
+@overload
+def clean_editor_js(definitions: None) -> None:
+    ...
```

Comment:
Re: warnings from coverage.py/Codecov, for the time being `@overload` could be excluded from coverage:
```patch
diff --git a/setup.cfg b/setup.cfg
--- a/setup.cfg	(revision f1c00de75df0244e887746c32cc89fbe49ec9772)
+++ b/setup.cfg	(date 1672327107349)
@@ -14,6 +14,7 @@
     raise NotImplementedError
     return NotImplemented
     if TYPE_CHECKING:
+    @overload
 
 [tool:pytest]
 addopts = -n auto --record-mode=none --ds=saleor.tests.settings --disable-socket
```

Related ticket: https://github.com/nedbat/coveragepy/issues/970